lib/node_modules/@stdlib/blas/ext/base/sdsapxsumpw/manifest.json

```diff
@@ -1,40 +1,40 @@
 {
 	"options": {},
 	"fields": [
-		{
-			"field": "src",
-			"resolve": true,
-			"relative": true
-		},
-		{
-			"field": "include",
-			"resolve": true,
-			"relative": true
-		},
-		{
-			"field": "libraries",
-			"resolve": false,
-			"relative": false
-		},
-		{
-			"field": "libpath",
-			"resolve": true,
-			"relative": false
-		}
+	  {
+	   "field": "src",
+	   "resolve": true,
+	   "relative": true
+	  },
+	  {
+	   "field": "include",
+	   "resolve": true,
+	   "relative": true
+	  },
+	  {
+	   "field": "libraries",
+	   "resolve": false,
+	   "relative": false
+	  },
+	  {
+	   "field": "libpath",
+	   "resolve": true,
+	   "relative": false
+	  }
 	],
 	"confs": [
-		{
-			"src": [
-				"./src/sdsapxsumpw.c"
-			],
-			"include": [
-				"./include"
-			],
-			"libraries": [
-				"-lm"
-			],
-			"libpath": [],
-			"dependencies": []
-		}
+	  {
+	   "src": [
+		"./src/sdsapxsumpw.c"
+	   ],
+	   "include": [
+		"./include"
+	   ],
+	   "libraries": [
+		"-lm"
+	   ],
+	   "libpath": [],
+	   "dependencies": []
+	  }
```

Comment:
This file has incorrect indentation, you may setup editor config extension in VSCode.